nav2_bringup/package.xml

```diff
@@ -10,16 +10,12 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_depend>rclcpp</build_depend>
+  <build_depend>rclcpp</build_depend>  
```

Comment:
whitespace, fixing

Reply:
so should rclcpp be in our out? There's no C++ here so I had removed it but then added it back. I think this should be the minimal set of dependencies, not leaving anything in here that's not needed, including rclcpp in this case.

Reply:
I think it shouldn't be necessary, but I'd leave it in for the Crystal release to be safer. Having it in is harmless.

Reply:
I'd leave rclcpp unless your future plans for this package will never include any non-launch code. It's one of those basic needs that people will frequently forget to add (and check for) that could burn us later